### PR TITLE
Update PortalDataset.yaml

### PR DESCRIPTION
**Pull request by anngvu**
Here's a mockup of what this might look like in the UI (maybe fancier than a link, should be an icon specific to the viewer). This would be useful for known viewers that can be linked to via a constant accession/url, such as cBioPortal, GEO, CELLxGENE.

Update: Suggestion for a better term other than "externalViewer" welcome.

image

### CODE REVIEW

**Comment by anngvu** [comment]
@allaway Yeah, I know there's the attribute-per-viewer option, and I have reasons for not liking it. But anyway, there wasn't a pros and cons written down anywhere that I can find, so created issue https://github.com/nf-osi/nf-metadata-dictionary/issues/455 for more formal discussion/design documentation.

Good to know about the identifiers prefix!

**Comment by jaybee84** [comment]
Some alternate options for `external viewer` : 
1. `visualize data on` -> does not work for link to GEO/SRA (that might be okay since this is specifically for viewers)
2. `explore data` 
3. `explore insights` -> does not work for link to GEO/SRA (that might be okay since this is specifically for viewers)
4. `additional portal`
5. `More Details`
6. `Additional resources`
7. `Extra Content`
8. `Further Details`

4-8 are more generic and may work for links other than visualization portals.

**Review by cconrad8 — APPROVED** [review]

**Comment by allaway** [comment]
@jaybee84 I likke `Explore Insights` and `visualize data on`! If we start linking to other _raw data_ I think we should categorize that separately anyway, so probably not too much of a problem (does not work for link to GEO/SRA (that might be okay since this is specifically for viewers)

File: modules/Template/PortalDataset.yaml [hunk 1]
```diff
@@ -72,6 +72,10 @@ classes:
       series:
         range: string
         required: false
+      externalViewer:
```
Comment (allaway) [inline modules/Template/PortalDataset.yaml:75]:
```suggestion
      visualizeDataOn:
```

Suggesting from @jaybee84

**Pull request merged**